web/client/components/geostory/media/Map.jsx

```diff
@@ -61,7 +63,18 @@ export default compose(
                 id: `media-${id}`
             }} // if map id is passed as number, the resource id, ol throws an error
             layers={layers}
-            options={applyDefaults(options)}
+            options={
+                // mouseWheelZoom is enabled only if inlineEditing is active and zoomControl too
```

Comment:
You should never activate mouseWheel interaction, even in edit mode. you could activate it using a switch, at least.

Reply:
as written and requested in the issue #4501, when inline editing is activated and zoomContorl is enabled then activate also the MouseWheelZoom interaction.

probably i just need to change the actual condition where it does it

Reply:
Ok, so this becomes a suggestion